### DISCUSSION TITLE
Error exeption....

### DISCUSSION THREAD

**Discussion by mance819** [Chia-Network/chia-blockchain#5127] [Support]
ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call

This error repeats itself every minute in the log and idk what's causing it. Has anyone solved this?

**Comment by rennster** [comment]
Same problem here.

Win 10 ver 20H2
Chia version 1.1.5
Port 8444 forwarded, uPNP disabled

**Comment by BrandtH22** [comment]
@mance819 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
